### PR TITLE
Update sqltca script

### PR DESCRIPTION
**Pull request by maximaexchen**
I had errors by only removing the t3ver_ columns in the "ext_tables.sql"
Had to set the correct versioning handling in table TCA

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -34,4 +34,5 @@
 *.tdo
 _minted*
 .texpadtmp/
+.idea
```
Comment (oliverklee) [inline .gitignore]: Could you please remove this change from this PR (as it's not related)? Thanks!

File: extbase-best-practices/sql-tca.tex [hunk 2]
```diff
@@ -10,7 +10,7 @@ \section{Remove unneeded default-generated columns}
 \begin{description}
   \item[hidden, starttime, endtime] if you do not need this restrictions
   \item[sys\_language\_uid, l10n\_parent, l10n\_diffsource, KEY language] if the records in this table will never be translated
-  \item[t3ver\_*] if you do not need versioning or workspaces for this table
+  \item[t3ver\_*] if you do not need versioning or workspaces for this table. Also set \texttt'versioningWS'=>false,, in \textttTCA/tx_extension_domain_model_table.php
```
Comment (oliverklee) [inline extbase-best-practices/sql-tca.tex]: The correct syntax for texttt would be with curly braces:

\texttt{foo ... bar}

This also allows you to have spaces in the monospace parts.

Comment (maximaexchen) [reply extbase-best-practices/sql-tca.tex]: I changed it.

**Comment by oliverklee** [comment]
Merged. Thanks for contributing, @maximaexchen! 🍪

**Pull request merged**